common-rendering/src/components/FirstPublished.tsx

```diff
@@ -30,25 +32,27 @@ const FirstPublished = ({
 				}
 			`}
 		>
-			<time
-				dateTime={publishedDate.toISOString()}
-				data-relativeformat="med"
-				css={css`
-					color: ${neutral[46]};
-					font-weight: bold;
-					margin-right: ${space[2]}px;
-				`}
-			>
-				{timeAgo(firstPublished)}
-			</time>
+			{!isPinnedPost && (
+				<time
+					dateTime={publishedDate.toISOString()}
+					data-relativeformat="med"
+					css={css`
+						color: ${neutral[46]};
+						font-weight: bold;
+						margin-right: ${space[2]}px;
+					`}
+				>
+					{timeAgo(firstPublished)}
+				</time>
+			)}
 			<span
```

Comment:
Not really sure where this change needs to happen (I'm guessing here but could well be wrong!) but it seems there's too much padding in front of the From time
<img width="742" alt="Screenshot 2022-03-17 at 16 27 04" src="https://user-images.githubusercontent.com/1336821/158847922-7d9060e3-b09e-4026-bf6b-15421edc5327.png">

Reply:
This is done